### DISCUSSION TITLE
Zigbee2MQTT non trova dispositivi aqara

### DISCUSSION THREAD

**Discussion by Sabream** [indomus/forum#2358] [HUB: Home Assistant]
Ciao a tutti, sono nuovo della community e mi sto cimentando da qualche giorno con un'installazione HA su un minipc con debian e docker.

Ho seguito tutte le varie guide relative all'installazione di HA, HACS, MQTT e Zigbee2mqtt ma sicuramente avrò commesso qualche errore in quanto nell'interfaccia web non risultano i vari dispositivi aqara che ho.
Premetto che non ho l'hub aqara ma soltanto il dongle zigbee 3.0 installato come primario.

Ho notato nella cartella di installazione di ha tutte le sottocartelle e i files hanno un lucchetto (screen https://imgur.com/a/SPFbymG)
Può causare questo il problema di pairing?

**Comment by indomus** [comment]
https://indomus.it/guide/integrare-componenti-zigbee-su-home-assistant-via-zigbee2mqtt/

**Reply by Sabream** [reply]
Grazie per la risposta, è proprio la guida che ho seguito alla lettera!
Solo che nell'interfaccia web di Zigbee2mqtt non vedo nulla...

Anche quando nella guida si parla di:
"In primis, prima di iniziare a utilizzare attivamente ZigBee2MQTT, è importante recarsi tramite interfaccia web alla sezione “impostazioni” e decidere quale canale radio utilizzare."
io nelle impostazioni non vedo assolutamente nulla:
![zhgare](https://user-images.githubusercontent.com/13134262/220639791-85d0a2d0-d745-4343-b072-75b7f52d4f82.JPG)

Ho provato a riavviare i vari docker e anche il minipc senza successo per il momento

**Reply by indomus** [reply]
provato con altro browser?

**Reply by Sabream** [reply]
Incredibile come uno cerchi ogni cosa cosmplicata e poi non provi la più semplice...
Ho sicuramente qualche plugin di firefox che impedisce il corretto funzionamento dell'interfaccia web zb2mqtt e con chrome è andata liscia!

Grazie mille!
